modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/freelist/io/PagesListMetaIo.java

```diff
@@ -51,7 +52,7 @@
      * @param ver Page format version.
      */
     private PagesListMetaIo(int ver) {
-        super(T_PAGE_LIST_META, ver);
+        super(12, ver, FLAG_AUX);
```

Comment:
So yeah, please introduce a constant. Why is it 12? We could start numbering them from 1